setup.py

```diff
@@ -13,7 +13,7 @@
 
 if 'install' in sys.argv:
 	# naively guess where the autostart .desktop file should be installed
-	if '--prefix' in sys.argv or '--home' in sys.argv:
+	if any(x.startswith('--prefix=') for x in sys.argv) or '--home' in sys.argv:
```

Comment:
You probably want to keep the `--prefix` check (after this change `setup.py --prefix /usr` will not be recognized anymore)

Reply:
For backward compatibility? Good idea.
(Otherwise it's a really strange way to write a long option)